client/main.lua

```diff
@@ -281,19 +284,18 @@ CreateThread(function()
         local sleep = 1000
         if LocalPlayer.state.isLoggedIn then
             sleep = 100
-
             local entering = GetVehiclePedIsTryingToEnter(cache.ped)
             local carIsImmune = false
             if entering ~= 0 and not isBlacklistedVehicle(entering) then
-                sleep = 2000
+                sleep = 500
```

Comment:
What's the rationale to reduce the sleep time?

Reply:
> Being able to drive when you don't have the keys for the first 2 seconds of the loop

Reply:
It waits too long to check if you can drive or not allowing you to drive for a short period of time before shutting off the car. Reducing it to 500 seems effective enough to stop this behaviour